lib/src/sql/data.rs

```diff
@@ -21,6 +21,7 @@ use std::fmt::{self, Display, Formatter};
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash)]
 pub enum Data {
 	EmptyExpression,
+	UnsetExpression(Vec<Idiom>),
```

Comment:
Just a nit-pick, could you move this down a line, so that it is in between `SetExpression`, and `PatchExpression`?

Reply:
Done :+1: